docs/strategies/hedging.md

```diff
@@ -147,6 +337,110 @@ With this configuration, the hedging strategy:
 - Initiates a maximum of `4` executions. This includes initial action and an additional 3 attempts.
 - Allows the first two executions to proceed in parallel, while the third and fourth executions follow the fallback mode.
```

Comment:
@martintmk At the _fallback mode_ section you have stated:
> In fallback mode, the `Delay` value should be less than `TimeSpan.Zero`.

And here we are using `InfiniteTimeSpan`. I'm not sure that at glance it is obvious why `ystem.Threading.Timeout.InfiniteTimeSpan < TimeSpan.Zero` is true.

I think it might make sense to add some explanation here. Do you agree?

Reply:
Agree, the behavior will be the same whenever you provide `System.Threading.Timeout.InfiniteTimeSpan` or any value lesser than 0.

Maybe System.Threading.Timeout.InfiniteTimeSpan wasn't the good choice as it suggests infinite waiting, i.e. fallback.

Reply:
In the next PR I will replace `System.Threading.Timeout.InfiniteTimeSpan` to `TimeSpan.FromSeconds(-1)`